packages/editor-ui/src/components/CredentialPicker/CredentialPicker.vue

```diff
@@ -0,0 +1,160 @@
+<template>
+	<div>
+		<div v-if="credentialOptions.length > 0" class="dropdown">
+			<CredentialsDropdown
+				:credential-type="credentialType"
+				:credential-options="credentialOptions"
+				:selected-credential-id="selectedCredentialId"
+				@credential-selected="onCredentialSelected"
+				@new-credential="createNewCredential"
+			/>
+
+			<n8n-icon-button
+				icon="pen"
+				type="secondary"
+				:class="{
+					edit: true,
+					invisible: !selectedCredentialId,
```

Comment:
I used `visibility: hidden` because otherwise it causes the dropdown to jump and resize:


https://github.com/n8n-io/n8n/assets/10324676/01d474d8-a39f-4d31-82dc-30b05cbb4802

compared to:


https://github.com/n8n-io/n8n/assets/10324676/e9dff477-0ca7-4d21-b6b5-4cfdfd9ce6d7

Reply:
Yeah, in that case we'll need a `min-width` on the dropdown or something to hold the size. But ok, I'm fine with both.